### DISCUSSION TITLE
Pull data from someone else's WeatherFlow?

### DISCUSSION THREAD

**Discussion by kchiem** [briis/hass-weatherflow#20] [Q&A] [answered]
Is there a way to use this integration with someone else's WeatherFlow?  I was able to do that with the previous incarnation of this integration (before the rename).

**Comment by briis** [comment]
This integration uses exactly the same authentication method as the previous one. WeatherFlow has changed their policy a year ago, so that people need to own a WeatherFlow station to get a useful Personal Token.
Before that, there was a general available Developer Token, that could be used together with all Station marked as Public - and as far as I know, this Token stills works. So if you use the same key as you used before it should still work - just note that I do not know for how long, as WeatherFlow might decide to make the Developer Key obsolete.

**Reply by kchiem** [reply]
What's the developer token?  I don't remember having to enter this with the previous version.

**Comment by briis** [comment, marked as answer]
I am positive you have entered a Token also in the old Integration, as this has always been they way these works.

Here is the Developer Token `20c70eae-e62f-4d3b-b3a4-8586e90f3ac8`, but just remember, there is no guarantee how long this work.

**Reply by kchiem** [reply]
I honestly don't remember entering a token before, but thank you for the developer token.  I was able to get the new integration working with it.
